assets/src/content/Routes.tsx

```diff
@@ -12,6 +12,7 @@ const Routes: React.FC = () => {
   }
   let { url } = match;
   url = url.endsWith("/") ? url.slice(0, -1) : url;
+  url = ""; // what TODO: jingram?
```

Comment:
@dagoof it's because the content routes need to look up their prefix. The content routes are just under the root

`path=""` below:

```
<Route path="" strict>
  <Content.Routes />
</Route>
```

But I thought they may move to something like `/content/terms`. It's fine to remove `url` from this render component—I probably just kept it in to match the way the other `Routes.tsx` files did it because they definitely need it.